packages/playground/tests/frontend_selenium/utils/utils.py

```diff
@@ -176,16 +176,29 @@ def randomize_public_ipv4():
     ip_subnet = ip + '/' + random.choice(['26', '27', '28', '29'])
     return ip_subnet, ip
 
+
+def convert_to_scaled_float(number):
+    str_number = str(number)
+    if '.' in str_number:
+        decimal_index = str_number.index('.')
+    else:
+        decimal_index = len(str_number)
+    divisor = 10 ** decimal_index
+    scaled_number = number / divisor
+    return scaled_number
+
+
+
 def byte_converter(value):
```

Comment:
is suggest to define the unit, and the value before the if condition to make it readable 
```ts
 unit = value[-1].upper()  
 number_str = value[:-1].strip()
```
then use it again

Reply:
Done